_docs/performance-tuning/drill-metastore/010-using-drill-metastore.md

```diff
@@ -364,7 +406,10 @@ apache drill> ANALYZE TABLE dfs.tmp.lineitem DROP METADATA;
 1 row selected (0.291 seconds)
 ```
 
-### Collecting metadata for specific set of columns
+### Collect metadata for specific set of columns
+
+Next lets gather metadata for a subset of the columns in the `lineitem` table. You would do this when you know
+ which columns will be present in filter conditions or data size or columns count is too large.
```

Comment:
A bit awkward. Maybe something like: "You would do this to allow Drill to optimize `WHERE` conditions on certain columns. Also, if file size or the number of columns grows large, it can take too long to gather all statistics. Instead you can speed up analysis by gathering statistics only for selected columns: those actually used in the `WHERE` clause."